### PR TITLE
Vote all classifications

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds ability to up or downvote all classifications.

### CODE REVIEW

File: static/js/components/SourceTable.jsx [hunk 1]
```diff
@@ -334,9 +356,49 @@ const RenderShowClassification = ({ source }) => {
     });
   };
 
+  const addVotes = (vote) => {
+    source.classifications?.forEach((c) => {
+      dispatch(sourceActions.addClassificationVote(c.id, { vote })).then(
+        (result) => {
+          if (result.status === "success") {
+            dispatch(showNotification("Vote registered"));
```
Comment (bfhealy) [inline static/js/components/SourceTable.jsx:364]: Displaying "Vote registered" for each classification is accurate but can temporarily take up a lot of screen space. Is it worth adding an array to keep track of each "success" and, if votes are uniformly successful, displaying a single "Votes registered" notification?

File: static/js/components/SourceTable.jsx [hunk 2]
```diff
@@ -347,6 +409,20 @@ const RenderShowClassification = ({ source }) => {
...
+            >
+              <ThumbDown color={downvoteColor} />
+            </Button>
+            <Button
+              key={source.id}
+              id="up_vote"
+              onClick={() => addVotes(upvoteValue)}
+            >
+              <ThumbUp color={upvoteColor} />
+            </Button>
```
Comment (bfhealy) [inline static/js/components/SourceTable.jsx:425]: I was experimenting with the thumbs up/down box for all classifications. Might the below box reduce unintentional clicks of the delete-all button?
Screen Shot 2023-01-11 at 10 55 34 AM

**Review by bfhealy — APPROVED** [review]
Thanks @mcoughlin, this feature works precisely as I hoped it would. Just a few thoughts about what we display on the frontend.

File: static/js/components/SourceTable.jsx (hunk 2 above)
Comment (mcoughlin) [reply static/js/components/SourceTable.jsx:425]: @bfhealy It's a bit up to you. I expect you will be a power user of this feature, so whatever you think is best, feel free to push it.

File: static/js/components/SourceTable.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/SourceTable.jsx]: @bfhealy I pushed a change for this.

File: static/js/components/SourceTable.jsx (hunk 2 above)
Comment (bfhealy) [reply static/js/components/SourceTable.jsx]: @mcoughlin Just pushed the new box - I think it will be beneficial to keep the three buttons more separated.

**Pull request merged**